test/carts/deleteBillingAddressTest.js

```diff
@@ -39,34 +40,38 @@ describe('Magento deleteBillingAddress', () => {
         });
 
         it('successfully returns a cart after the billing address was deleted', () => {
-            const args = {
-                id: '12345-7'
-            };
+
+            let specs = specsBuilder();
```

Comment:
@herzog31 can't follow what you mean here.

Reply:
Yes I agree, someone would have to check what is the problem. I didn't want to duplicate all those tests because with graph ql this is not going to be need it anymore (I mean the spec thing). I understood what you mean by having different names but I am not sure what you suggest as a solution :)

Reply:
What about adding a suffix to the test name? You could wrap the `it` closure with the spec thing.

Reply:
didn't know that's possible. yes that can be an improvement.